BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -714,8 +715,13 @@ extension GiniBankNetworkingScreenApiCoordinator: SkontoCoordinatorDelegate {
 }
 
 extension GiniBankNetworkingScreenApiCoordinator {
-    private func setTransactionDocsDataToDisplay(with extractionResult: ExtractionResult) {
+    private func setTransactionDocsDataToDisplay(with extractionResult: ExtractionResult, for documentId: String) {
         transactionDocsDataCoordinator?.loadDocumentData = { [weak self] in
+            if let images = self?.transactionDocsDataCoordinator?
```

Comment:
[suggestion] I think is much better to move in a separate method the content of the closure for `transactionDocsDataCoordinator?.loadDocumentData = { [weak self] in`. It will improve visibility and readability. As well we will silence one of Sonarcloud's warnings -> "Refactor this code to not nest more than 2 closure expressions. "